airflow/jobs.py

```diff
@@ -1906,12 +1906,15 @@ def __init__(
         :param args:
         :param kwargs:
         """
+        if donot_pickle is None:
+            self.donot_pickle = configuration.getboolean("core", "donot_pickle")
```

Comment:
```suggestion
            self.donot_pickle = configuration.conf.getboolean("core", "donot_pickle")
```

Reply:
> Accessing configuration method 'getboolean' directly from the configuration module is deprecated. Please access the configuration from the 'configuration.conf' object via conf.getboolean

Reply:
din't know that, fixed that now. Also I now did remove the of statement and replaced this by:
```
self.donot_pickle = donot_pickle or configuration.conf.getboolean("core", "donot_pickle")
```

Reply:
well did revert that, because `None or False == False`, `None or True == True`. But if the argument is filled this will happen: `False or False == False`, `False or True == True`. my bad ;)